app/webpack/dev.config.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+module.exports = () => {
+  if (!process.env.PROXY_PATH) {
+    console.log('========== Specify the PROXY_PATH variable in the .env file =========');
+    process.exit(1);
+  }
+  return {
+    devtool: 'eval-sourcemap',
+    mode: 'development',
+    module: {
+      rules: [
+        {
+          test: /\.(sa|sc|c)ss$/,
+          use: [
+            'style-loader',
+            {
+              loader: 'css-loader',
+              options: {
+                modules: true,
+                importLoaders: 1,
+              },
+            },
+            'sass-loader',
+            {
+              loader: 'sass-resources-loader',
+              options: {
+                resources: path.resolve(__dirname, '../src/common/css/variables/**/*.scss'),
+              },
+            },
+          ],
+        },
+      ],
+    },
+    plugins: [
+      new webpack.HotModuleReplacementPlugin(),
+      new CleanWebpackPlugin({
```

Comment:
Can we move this plugin definition to the `base.config.js` ?
As I see it's absolutely the same with [this](https://github.com/reportportal/service-ui/pull/2211/files#diff-7c9b12d67fa5d63ba0049392cbed6b9cR64-R66).